projects/helper/tokenMapping.js

```diff
@@ -43,6 +43,30 @@ const fixBalancesTokens = {
   ozone: {
     // '0x83048f0bf34feed8ced419455a4320a735a92e9d': { coingeckoId: "ozonechain", decimals: 18 }, // was mapped to wrong chain
   },
+
+  provenance: {
```

Comment:
please revert this change and let us know if there are any assets still missing. We have moved this logic to our coins server